src/test/java/org/broadinstitute/hellbender/tools/gvs/common/WeightedSplitIntervalsIntegrationTest.java

```diff
@@ -62,7 +54,6 @@ public void testNoLossRealisticWgs() {
 
         final int scatterCount = 100;
         final File outputDir = createTempDir("output");
-        final Interval interval = new Interval("chr20", 1000000, 2000000);
```

Comment:
this variable in an existing wasn't being used which may or may not be concerning?

Reply:
Probably just a cut and paste from testNoLossSimple

Reply:
Good spot, I think you're probably right.